### ISSUE TITLE
Adding new table to the models section

### ISSUE THREAD

**Issue by haysconun** [flatironinstitute/blastoSPIM_web#16]
immediately below the current table in the models section, could you add the current table 2 (in the pdf that I sent you yesterday)

The description of the table would be "Performance results on moderate SNR Images per developmental stage for both the Stardist-3D early embryo model and the Stardist-3D late blastocyst model. This test set includes test images from both BlastoSPIM 1.0 and 2.0. Model hyperparameters were fixed for both models across all stages. The model trained on BlastoSPIM 2.0 starts to outperform the model trained on BlastoSPIM 1.0 during the transition between the 32-cell stage and the 64-cell stage."
